src/Configuration.cpp

```diff
@@ -50,3 +50,29 @@ string imu_aceinna_openimu::to_string(Configuration::Orientation const& o)
 {
     return axisToUser(o.forward) + axisToUser(o.right) + axisToUser(o.down);
 }
+
+bool Configuration::operator ==(Configuration const& other) const {
+    return !(*this != other);
+}
+
+bool Configuration::operator !=(Configuration const& other) const {
+    return
+        needsReset(other) ||
+        other.periodic_packet_rate != periodic_packet_rate ||
+        other.periodic_packet_type != periodic_packet_type;
```

Comment:
why aren't these included in the needsReset? I would assume that if the packet_rate or packet_type configurations changed, it would need a reset, even if, from what I understood, the orogen doesn't touch these parameters.

Reply:
Those can dynamically be changed. The component actually sets them on start and resets them to zero on stop.